asct-b/bone-marrow-and-blood.md

```diff
@@ -8,13 +8,13 @@ Anatomical Structures, Cell Types, plus Biomarkers (ASCT+B) tables aim to captur
 | :------------- |:-------------|
 | **Creator(s):** | Muzlifah A. Haniffa; Laura Jardine; Sarah A. Teichmann |
 | **Creator ORCID:** | 0000-0002-3927-2084; 0000-0003-4495-8205; 0000-0002-6294-6366 |
-| **Project Lead:** | Katy Börner |
+| **Project Lead:** | Katy B&ouml;rner |
 | **ORCID:** | https://orcid.org/0000-0002-3321-6137 |
 | **Creation Date:** | 2021-03-12 |
 | **License:** | Creative Commons Attribution 4.0 International ([CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)) |
 | **Publisher:** | HuBMAP |
 | **Funder:** | National Institutes of Health |
 | **Award Number:** | OT2OD026671 |
-| **Publication:** | *Nature Cell Biology* |
+| **Publication:** | *Nature Cell Biology Paper* |
```

Comment:
@lecross -- This is weird: The name of the journal is just "Nature Cell Biology". Whatever.